kolibri/core/assets/src/kolibri_app.js

```diff
@@ -3,8 +3,24 @@ import router from 'kolibri.coreVue.router';
 import Vue from 'kolibri.lib.vue';
 import store from 'kolibri.coreVue.vuex.store';
 import heartbeat from 'kolibri.heartbeat';
+import * as Sentry from '@sentry/browser';
+import logger from 'kolibri.lib.logging';
 import KolibriModule from 'kolibri_module';
 
+export const logging = logger.getLogger(__filename);
+
+if (global.sentryEnabled) {
```

Comment:
Can avoid bundling this into the default frontend bundle by making a separate bundle, and conditionally loaded tag. Will still be able to access the global kolibri Vue through `kolibri.lib.vue`.